### DISCUSSION TITLE
missing *.lib files for windows c shared libraries

### DISCUSSION THREAD

**Discussion by xjjius** [epam/Indigo#680] [General]
I used to download the compiled c libraries from official website, which works perfect for my own project.

However, I found that there are no *.lib files for c shared libraries after version 1.4.0. And I got the link error when I compile my c project under Windows, as expected. 

Could you please add *.lib files of c shared libs for Windows X32/X64 build?

**Comment by SunFellow** [comment]
Thank you for pointing this out!
This is a real shortcoming in our delivery artifacts after 1.4.0: we only collect binary outputs and forgot about making the self-sufficient Indigo Toolkit delivery. The downloads now miss both Header files and Import Library files.
